mdpow/workflows/dihedrals.py

```diff
@@ -13,47 +13,58 @@
 depending on the desired results. Details of the completely automated workflow
 are discussed under :func:`~mdpow.workflows.dihedrals.automated_dihedral_analysis`.
 
+Atom indices obtained by :func:`get_atom_indices` are 0-based,
+atom index labels on the molecule in the plots are 0-based,
+but atom `names` in plots and file names are 1-based.
+
 .. autodata:: SOLVENTS_DEFAULT
     :annotation: = ('water', 'octanol')
 .. autodata:: INTERACTIONS_DEFAULT
     :annotation: = ('Coulomb', 'VDW')
 .. autodata:: SMARTS_DEFAULT
     :annotation: = [!#1]~[!$(*#*)&!D1]-!@[!$(*#*)&!D1]~[!#1]
+.. autodata:: PLOT_WIDTH_DEFAULT
+    :annotation: = 190
 .. autofunction:: automated_dihedral_analysis
 .. autofunction:: build_universe
 .. autofunction:: rdkit_conversion
-.. autofunction:: dihedral_indices
-.. autofunction:: dihedral_groups
+.. autofunction:: get_atom_indices
+.. autofunction:: get_bond_indices
+.. autofunction:: get_dihedral_groups
+.. autofunction:: get_paired_indices
 .. autofunction:: dihedral_groups_ensemble
 .. autofunction:: save_df
-.. autofunction:: periodic_angle
+.. autofunction:: periodic_angle_padding
 .. autofunction:: dihedral_violins
-.. autofunction:: plot_violins
+.. autofunction:: build_svg
+.. autofunction:: plot_dihedral_violins
 
 """
 
 import os
+import logging
 import pathlib
+
 import numpy as np
 import pandas as pd
-
+import MDAnalysis as mda
+from MDAnalysis.topology.guessers import guess_types
 from rdkit import Chem
-
-import seaborn as sns
+from rdkit.Chem import rdCoordGen
+from rdkit.Chem.Draw import rdMolDraw2D
 import matplotlib.pyplot as plt
+import seaborn as sns
+import pypdf
+import cairosvg
+import svgutils
+import svgutils.compose
+import svgutils.transform
 
 import mdpow
 from mdpow.analysis.dihedral import DihedralAnalysis
```

Comment:
see PR #257

Reply:
@orbeckst I am looking over it now and then I will merge it, but I will have to manually merge the conflicts since my commit rearranged the inputs by category as outlined in PEP8.

Reply:
- whenever I merge `develop` into this PR

Reply:
sure, just merge develop into your PR and then manually resolve the conflicts

Reply:
once this PR is done, we can also consider to just run `black` on everything and forget about formatting